runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -143,11 +143,22 @@ default boolean screenshotBossKills()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "gauntletLoot",
+		name = "Screenshot Gauntlet Loot",
+		description = "Configures whether or not screenshots are automatically taken of Gauntlet Loot",
+		position = 10
+	)
+	default boolean screenshotGauntletLoot()
```

Comment:
I think this config is not needed since we can use the `screenshotRewards()` config that already exists.

Reply:
Gotcha thanks. Didn't even notice that happened 

Fixed adcb9e662a4dd734341b4a4cec2564dd8fdd4165

Reply:
Thanks, added it to the rewards config